agimus_controller/agimus_controller/ocp_param_base.py

```diff
@@ -7,15 +7,20 @@
 class OCPParamsBaseCroco:
     """Input data structure of the OCP."""
 
-    # Data relevant to solve the OCP
-    dt: float  # Integration step of the OCP
-    horizon_size: int  # Number of time steps in the horizon
-    solver_iters: int  # Number of solver iterations
+    # Data relevant to solve the OCP.
+    dt: float  # Integration step of the OCP.
+    horizon_size: int  # Number of time steps in the horizon.
+    solver_iters: int  # Number of solver iterations.
+    horizon_dts: list[
```

Comment:
Do you plan to convert `dt_factor_n_seq` into `horizon_dts` here ?
Or is this remaining from a previous iteration of your implementation ?

Reply:
remaining from the previous iteration, I did not yet propagate the API changes.